ovs/dal/hybrids/storagedriver.py

```diff
@@ -62,8 +65,16 @@ def _status(self):
         """
         Fetches the Status of the Storage Driver.
         """
-        _ = self
-        return None
+        vpool_status = self.vpool.status
```

Comment:
This is not right, right?
The vpool can be installing a different storagedriver